docs/adr/0004-lists-returned-with-root-json-object

```diff
@@ -0,0 +1,55 @@
+# 0004 - Lists returned with root JSON object
```

Comment:
I think this could be clearer about the decision. Maybe "Always return a JSON object for JSON responses" or "Never return a list at the root of a JSON response" or similar.

Reply:
Agreed, I was struggling naming this. I'm happy with "Always return a JSON object for JSON responses"